buildlib/template.pc.in

```diff
@@ -1,7 +1,7 @@
 prefix=@CMAKE_INSTALL_PREFIX@
 exec_prefix=${prefix}
```

Comment:
I think that these two line above can be deleted.

Reply:
That would change the interface (removing those variables). I fear that might cause regressions. I'd rather keep this risk free. (BTW, `exec_prefix=` was unreferenced already before my patch.)

Reply:
> That would change the interface (removing those variables). I fear that might cause regressions. I'd rather keep this risk free. (BTW, `exec_prefix=` was unreferenced already before my patch.)

I don't know about which regressions you are talking. Both prefix and exec_prefix are variables in pckg-config DSL, they are mpt exported.

Thanks

Reply:
All variables are available to the CLI. I'm thinking *someone* might be doing this:

```
$ pkg-config --variable=prefix librdmacm
/nix/store/g2wpfpy5gban3jilzvvfv7aapl0g4b53-rdma-core-37.1
```

That's why I'd prefer to separate bugfix from clean-up and not to both in one commit/PR, in case it causes problems for someone.

Reply:
You already broke such use case, because before your change, user could issue --define-variable and expect that prefix will be picked. It is not the case anymore.

This is why I think that if we take your PR, not-used variables should be removed too.

Reply:
Oh, I didn't know about the `pkg-config --define-variable=NAME=VALUE` option. I've always thought of *.pc files as constants. I don't understand when that feature is useful though. Why ask pkg-config for a variable if the plan is to override it? I don't think it's a good idea to make assumptions about if/how some variables in a .pc file is derived from others (i.e. assume that overriding `prefix=` would work and create correct paths for other variables). In fact, assuming that e.g. includedir= is a subtree of prefix= isn't always correct, and the reason I had to write this patch in the first place.

However, I would like to fix the rdma-core *.pc files more than I'd like to argue about this point, so I've added a commit to remove the variables.